starboard/android/shared/audio_track_audio_sink_type.cc

```diff
@@ -50,8 +51,10 @@ const SbTime kMaxDurationPerRequestInTunnelMode = 16 * kSbTimeMillisecond;
 
 const size_t kSilenceFramesPerAppend = 1024;
 
-const int kMaxRequiredFrames = 16 * 1024;
-const int kRequiredFramesIncrement = 2 * 1024;
+const int kMaxRequiredFramesLocal = 16 * 1024;
+const int kMaxRequiredFramesRemote = 32 * 1024;
+const int kMaxRequiredFrames = kMaxRequiredFramesRemote;
+const int kRequiredFramesIncrement = 4 * 1024;
```

Comment:
nit: For my understanding, why do you increase `kRequiredFramesIncrement`?

Reply:
It's to reduce  the test running time, as the maximum for remote device was set to 32k, and the loss in precision of the test results is acceptable.